### ISSUE TITLE
Item aria label does not parse html

### ISSUE THREAD

**Issue by guywillis** [adaptlearning/adapt-contrib-accordion#132]
### Subject of the issue
Accordion item aria label does not allow html to be parsed through.

### Your environment
* FW / AAT master

### Expected behaviour
HTML should be allowed to parse through the Accordion item aria label
